src/microbe_stage/Microbe.cs

```diff
@@ -203,6 +203,8 @@ public MicrobeSignalCommand SignalCommand
     [JsonProperty]
     public int DespawnRadiusSquared { get; set; }
 
+    public int EntityWeight => organelles?.Count ?? 1;
```

Comment:
Entity weights should be JsonIgnore if they are computed like this (even without a setter the json lib will write out this value uselessly into the json).

Also as a safety measure I would just throw an exception if the Microbe is not initialized yet before the entity weight is accessed, because it would be a pretty nightmare debugging situation to try to find out why the entity weight system is not working if we get reports it doesn't seem to work.

Reply:
can we get a setting to turn this off? since it doesn't have a large performance impact but would make hunting harder, it can already be difficult to hunt for those of us who need to use the lower entity limits.